test/e2e/rich-editor.cy.ts

```diff
@@ -0,0 +1,46 @@
+import { getIdContent } from "test/support/commands";
+
+context("RichEditor Component", () => {
```

Comment:
can I suggest we chagne "Print Console" to just "Print" and print that to a `<p>` below the editor? and then, we can test the represented content through this `<p>`...